### ISSUE TITLE
cdn can't load css and js, can it be changed to local?

### ISSUE THREAD

**Issue by 107295472** [exceptionless/Exceptionless#1056]

**Comment by niemyjski** [comment]
What error are you getting? We moved to a global cdn a few years ago that should work in every country.

**Comment by 107295472** [comment]
cdn.jsdelivr.net  This domain name is not good, there is also Google's

**Comment by niemyjski** [comment]
We moved off of a different cdn here: https://github.com/exceptionless/Exceptionless.UI/issues/195 due to issues. We'd be willing to accept a pr to bring it in internally. What country are you in?

**Comment by 107295472** [comment]
Can't you use a cdn?

**Comment by niemyjski** [comment]
Yes, we use cdn's, we just don't have the bandwidth to be changing them yearly due to countries randomly blocking valid world wide cdns. If you can open a PR for bringing the dependencies into a vendor bundle we would be more than happy to accept the pr.
